### DISCUSSION TITLE
How to connect to API?

### DISCUSSION THREAD

**Discussion by xKugeki** [dani-garcia/vaultwarden#3764] [Q&A] [answered]
Hey I tried to connect to my API from my Organisation.
I found the API Key under Organisation > Settings > Show API-Key

and got a **client_id**, **client_secret**, **scope** and **grant_type**.

But how do I connect to e.g hoppscotch?
I saw that vaultwarden/bitwarden uses OAuth2

So I need the following:
![image](https://github.com/dani-garcia/vaultwarden/assets/119607113/f79c0b3b-dad4-4e81-83a9-9874a9d5d546)

but where to find these URLs ? :)

**Comment by BlackDex** [comment, marked as answer]
Vaultwarden doesn't support the full public API. Only just enough for the Directory Connector to work.

**Reply by BlackDex** [reply]
Also, Vaultwarden doesn't support SSO. There is a PR for this, but nothing in a current version of Vaultwarden.

**Comment by xKugeki** [comment]
Alright thank you :)
